storage/simple_tree.hpp

```diff
@@ -64,6 +68,44 @@ class SimpleTree
         it->Sort(false);
   }
 
+  /// \brief Checks all nodes in tree to find an equal one. If there're several equal nodes
+  /// returns the first found.
+  /// \returns a poiter item in the tree if found and nullptr otherwise.
+  /// @TODO The complexity of the method is O(n). But the structure (tree) is built on the start of the program
+  /// and then activly on run time. This method should work at constant time.
```

Comment:
It's unclear for me how this method can work at constant time. Especially in presence of the previous sentence (about O(n)).